src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -4863,6 +4939,401 @@ resource pipeline_RerunETL 'Microsoft.DataFactory/factories/pipelines@2018-06-01
   }
 }
 
+//------------------------------------------------------------------------------
+// recommendations export pipeline
+// Triggered by dailyRecommendations trigger
+//------------------------------------------------------------------------------
+@description('Extracts Azure Advisor and custom recommendations from the Resource Graph API.')
+resource pipeline_ExecuteRecommendations 'Microsoft.DataFactory/factories/pipelines@2018-06-01' = {
+  name: '${recommendationsDataSet}_Execute'
```

Our pattern is to name these based on the source/destination. From our perspective, the source is the query files, not exactly ARG, so I'm thinking that might be the best source 🤔
```suggestion
  name: 'queries_ETL_ingestion'
```

---

Addressed with a different solution, due to the refactoring required by the queries processing generalization.